packages/core/strapi/lib/commands/build.js

```diff
@@ -24,7 +24,8 @@ module.exports = async ({ optimization, forceBuild = true }) => {
   await buildAdmin({
     buildDestDir,
     forceBuild,
-    isTSProject,
+    // TODO: Delegate the is-ts-project check to the admin
```

Comment:
What's the TODO for?

Reply:
You can remove that part and just check if we have a tsconfig.json file in the src/admin folder instead.

Reply:
But you'll have to know where to put the build folder however it's located inside the dist one only if the backend has a tsconfig.json file.

Reply:
Discussed & fixed in your branch/side, marking as closed & removing the todo